pages/vi/vi-github-issues.md

```diff
@@ -112,40 +126,23 @@ Additionally, creating a checklist will help others understand what you have don
 
 Some of these items are of course optional, but try to include as much useful information for others as possible in your checklists because nobody really wants to work on something that has already been completed by someone else.
 
-## Wait for Review
+## Delete the Branch after Staff review
```

Comment:
I prefer to make it as "Delete the Branch after your changes merged".

Reply:
@nadaa, thank you for your suggestion. 'Delete the Branch after changes merged' better corresponds with actual workflow, and 'Delete the Branch after Staff review' can be misleading. 
However, maybe as for the title, I will leave only 'Delete the Branch,' and explain when to delete in the paragraph itself.  What do you think?

Reply:
@svlesiv, it wil be better!